### PR TITLE
4.4.0

### PR DESCRIPTION
**Pull request by eromano**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [x] Tests for the changes have been added (for bug fixes / features)
> - [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [x] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
4.4.0

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: docs/upgrade-guide/README.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ Title: Upgrade guides
 
 Below are links to the upgrade guides notes for all released versions of ADF back to v2.6.0
 
+- [Upgrading from ADF v3.2 to v3.3](upgrade43-44.md)
+- [Upgrading from ADF v3.2 to v3.3](upgrade42-43.md)
```
Comment (arditdomi) [inline docs/upgrade-guide/README.md:10]: the last 5 entries are "Upgrading from ADF v3.2 to v3.3" + the 2 new ones

File: docs/upgrade-guide/upgrade43-44.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+Do not skip this task, if you want your application to be updated to a most recent version of ADF. 
+Upgrades of multiple versions of ADF cannot be done in one step only, but should follow the chain of sequential updates. 
+
+**Note:** the steps described below might involve making changes
+to your code. If you are working with a versioning system then you should
+commit any changes you are currently working on. If you aren't using versioning
+then be sure to make a backup copy of your project before going ahead with the
+upgrade.
+
+This guide explains how to upgrade your ADF v3.2 project to work with v3.3.
```
Comment (arditdomi) [inline docs/upgrade-guide/upgrade43-44.md:18]: same here

**Review by arditdomi — CHANGES_REQUESTED** [review]

**Review by DenysVuika — APPROVED** [review]
Besides comments above, looks fine

File: docs/upgrade-guide/README.md [hunk 3]
```diff
@@ -6,10 +6,12 @@ Title: Upgrade guides
 
 Below are links to the upgrade guides notes for all released versions of ADF back to v2.6.0
 
-- [Upgrading from ADF v3.2 to v3.3](upgrade41-42.md)
-- [Upgrading from ADF v3.2 to v3.3](upgrade40-41.md)
-- [Upgrading from ADF v3.2 to v3.3](upgrade39-40.md)
-- [Upgrading from ADF v3.2 to v3.3](upgrade37-38.md)
+- [Upgrading from ADF v4.4 to v4.4](upgrade43-44.md)
```
Comment (arditdomi) [inline docs/upgrade-guide/README.md:9]:
```suggestion
- [Upgrading from ADF v4.3 to v4.4](upgrade43-44.md)
```

**Review by arditdomi — CHANGES_REQUESTED** [review]

**Review by arditdomi — APPROVED** [review]

**Pull request merged**